x-pack/filebeat/module/rabbitmq/log/ingest/pipeline.json

```diff
@@ -0,0 +1,37 @@
+{
+  "description": "Pipeline for parsing RabbitMQ logs",
+  "processors": [
+    {
+      "grok" : {
+        "field" : "message",
+        "pattern_definitions" : {
+          "GREEDYMULTILINE" : "(.|\n)*",
+          "ERL_PID" : "\\<%{INT}+\\.%{INT}+\\.%{INT}+\\>"
+        },
+        "patterns" : [
+          "%{DATESTAMP:timestamp} \\[%{WORD:log.level}\\] %{ERL_PID:rabbitmq.log.pid} %{GREEDYMULTILINE:message}"
+        ],
+        "ignore_missing" : true
+      }
+    },
+    {
+      "date" : {
+        "field" : "timestamp",
+        "formats" : ["yy-MM-dd HH:mm:ss.SSS"],
+        {< if .convert_timezone >}"timezone": "{{ event.timezone }}",{< end >}
```

Comment:
@exekias Should we still allow to disable the time zone conversion? I don't really see an option to make it optional and with https://github.com/elastic/beats/pull/12120 it's not going to be a problem anymore for the tests.

Reply:
I think it would make sense to hardcode this for new modules, we can always add it later if we see a demand for it